### PR TITLE
Update for 3.13.2

### PR DESCRIPTION
**Pull request by zombieJ**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zy410419243** [comment]
标题 3.13.2，还以为倒退了...

**Comment by zombieJ** [comment]
> 标题 3.13.2，还以为倒退了...

好眼力 😄

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ timeline: true
 
 ---
 
+## 3.13.2
+
+`2019-02-07`
+
+- 🐞 Adjust font style to fix display issue in IE. [#14708](https://github.com/ant-design/ant-design/pull/14708) [@ashearer](https://github.com/ashearer)
+- 🐞 Fix Table ajax render not correct issue. [#14724](https://github.com/ant-design/ant-design/pull/14724) [@imhele](https://github.com/imhele)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: 这个按优先级应该提到第一个。

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,14 @@ timeline: true
 
 ---
 
+## 3.13.2
+
+`2019-02-07`
+
+- 🐞 调整字体样式以适配旧版 IE 下的展示问题。[#14708](https://github.com/ant-design/ant-design/pull/14708) [@ashearer](https://github.com/ashearer)
+- 🐞 修复 Table 异步数据的展示问题。[#14724](https://github.com/ant-design/ant-design/pull/14724) [@imhele](https://github.com/imhele)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 描述的也不够详细。

**Comment by zombieJ** [comment]
@afc163 check again: #14749

**Pull request merged**